README.md

```diff
@@ -14,11 +14,11 @@ documents, and Ruby on Rails APIs.
 
 ## Getting started
 
-See the [/doc/usage/README.md usage]() guide to set up Megadoc for your project.
+See the [/doc/usage/README.md usage](../master/doc/usage/README.md) guide to set up Megadoc for your project.
```

Comment:
they do work when rendered, http://megadoc.github.io/megadoc/index.html

to fix them in github/markdown and keep the rendered version functional, make them relative (megadoc will follow relative links):

`./doc/usage/README.md`